lxd/device/nic_physical.go

```diff
@@ -31,10 +31,11 @@ func (d *nicPhysical) validateConfig(instConf instance.ConfigReader) error {
 		"maas.subnet.ipv4",
 		"maas.subnet.ipv6",
 		"boot.priority",
+		"gvrp",
 	}
 
 	if instConf.Type() == instancetype.Container {
-		optionalFields = append(optionalFields, "mtu", "hwaddr", "vlan")
+		optionalFields = append(optionalFields, "mtu", "hwaddr", "vlan", "gvrp")
```

Comment:
@TimRots please can you remove this bit, it is not needed as you already added `gvrp` to optionalFields for all instance types above.

Reply:
@TimRots ah actually, refreshing my memory please can you leave this bit and remove line 34, as the vlan (and by extension gvrp) feature isn't available for VMs.